src/pages/Download.js

```diff
@@ -1,9 +1,238 @@
-import { SectionTitle } from '../common/Section'
+import {
+  GridSection,
+  GridSectionTitle,
+  Section,
+  SectionTitle,
+  WideHSection,
+} from '../common/Section'
+
+import { BulletList, BulletListInside } from '../common/List'
+import Button from '../common/Button'
+
+import { Tab, Tabs, TabList, TabPanel } from 'react-tabs'
+import 'react-tabs/style/react-tabs.css'
+
+import { isWinPhone, isIOS, isWindows, isMacOs } from 'react-device-detect'
+import { useState } from 'react'
+
+import logo_name from '../assets/images/logo_web_name.png'
```

Comment:
It fails to build on my end due to missing `logo_web_name.png`. Maybe you forgot to include it in the commit? :)

Reply:
Done